content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md

```diff
@@ -0,0 +1,651 @@
+  <a name="Subject"></a>
+  *Subject contains a reference to the object or user identities a role binding applies to.  This can either hold a direct API object reference, or a value for non-objects such as user and group names.*
+-->
+- **subjects** ([]Subject)
+  
+  Subjects 包含角色所应用到的对象的引用。
+  
+  <a name="Subject"></a>
+  *Subject 包含角色绑定所应用到的对象或用户标识的引用。这可以包含直接 API 对象的引用或非对象（如用户名和组名）的值。*
+  <!--
+  - **subjects.kind** (string), required
+    Kind of object being referenced. Values defined by this API group are "User", "Group", and "ServiceAccount". If the Authorizer does not recognized the kind value, the Authorizer should report an error.
+  - **subjects.name** (string), required
+    Name of the object being referenced. 
+  -->
+  - **subjects.kind** (string)，必选
+    
+    被引用的对象的种类。这个 API 组定义的值是 `User`、`Group` 和 `ServiceAccount`。
+    如果 Authorizer 未识别种类值，则 Authorizer 应报告一个错误。
+  
+  - **subjects.name** (string)，必选
+    
+    被引用的对象的名称。
+  <!-- 
+    - **subjects.apiGroup** (string)
+    APIGroup holds the API group of the referenced subject. Defaults to "" for ServiceAccount subjects. Defaults to "rbac.authorization.k8s.io" for User and Group subjects.
+  - **subjects.namespace** (string)
+    Namespace of the referenced object.  If the object kind is non-namespace, such as "User" or "Group", and this value is not empty the Authorizer should report an error.
+  -->
+  - **subjects.apiGroup** (string)
+    
+    APIGroup 包含被引用主题的 API 组。对于 ServiceAccount 主题默认为 ""。
+    对于 User 和 Group 主题，默认为 "rbac.authorization.k8s.io"。
+  
+  - **subjects.namespace** (string)
+    
+    被引用对象的命名空间。
+    如果对象种类是 "User" 或 "Group" 等非命名空间且该值不为空，
+    则 Authorizer 应报告一个错误。
+<!-- 
+## ClusterRoleBindingList {#ClusterRoleBindingList}
+ClusterRoleBindingList is a collection of ClusterRoleBindings
+
+<hr>
+
+- **apiVersion**: rbac.authorization.k8s.io/v1
+- **kind**: ClusterRoleBindingList
+-->
+## ClusterRoleBindingList {#ClusterRoleBindingList}
+
+ClusterRoleBindingList 是 ClusterRoleBindings 的集合。
+
+<hr>
+- **apiVersion**: rbac.authorization.k8s.io/v1
+
+- **kind**: ClusterRoleBindingList
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+
+  Standard object's metadata.
+
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>), required
+
+  Items is a list of ClusterRoleBindings
+-->
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+  
+  标准对象的元数据。
+
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>)，必选
+  
+  Items 是 ClusterRoleBindings 的列表。
+<!-- 
+## Operations {#Operations}
+
+<hr>
+-->
+## 操作 {#Operations}
+
+<hr>
+<!-- 
+### `get` read the specified ClusterRoleBinding
+#### HTTP Request
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/{name}
```

Comment:
This line is not (and no need to) translate, maybe it's better to remove from comment.